framework/doc/content/source/controls/MOOSEFMU.md

```diff
@@ -0,0 +1,145 @@
+# MOOSE FMU Interface
+
+The `MOOSEFMU` defines the `Moose2FMU` base class which contains the boilerplate needed to wrap a MOOSE simulation as a Functional Mock-up Unit (FMU). Users only need to implement their own `__init__` and `do_step` methods when deriving from `Moose2FMU`.
```

Comment:
what standard did you implement for FMU ? 2? 3? do we have the whole standard implemented or partiaL?